src/ApplicationResolver.php

```diff
@@ -98,7 +99,7 @@ class_exists($class, true);
      */
     private static function getProjectSearchDirs(string $namespace): array
     {
-        $reflection = new \ReflectionClass(\Composer\Autoload\ClassLoader::class);
+        $reflection = new \ReflectionClass(ClassLoader::class);
```

Comment:
```suggestion
        $reflection = new ReflectionClass(ClassLoader::class);
```

Could you add `use ReflectionClass;` ?

Reply:
Why? Not related to this PR.

Reply:
If you're at it please ...